packages/block-editor/src/components/block-alignment-matrix-toolbar/README.md

```diff
@@ -0,0 +1,72 @@
+# Alignment Matrix Toolbar
+The alignment matrix toolbar allows users to quickly adjust inner block alignment; this is in contrast to the alignment toolbar that aligns the frame block.
+
+![Button components](https://i.imgur.com/PxYkgL5.png)
+
+## Table of contents
+- [Alignment Matrix Toolbar](#alignment-matrix-toolbar)
+	- [Table of contents](#table-of-contents)
+	- [Design guidelines](#design-guidelines)
+		- [Usage](#usage)
+		- [Best practices](#best-practices)
+	- [Development guidelines](#development-guidelines)
+		- [Usage](#usage-1)
+		- [Props](#props)
+
+## Design guidelines
+
+### Usage
+
+The alignment matrix is a specialized tool, and it's used in the cover block.
+
+![Cover](https://i.imgur.com/nJjqen8.png)
+
+As an example, here's the matrix alignment tool in action.
+
+![center](https://i.imgur.com/0Ce1fZm.png)
+
+
+![rop_right](https://i.imgur.com/yGGf6IP.png)
+
+### Best practices
```

Comment:
It seems this section doesn't add much value to me for this component, should we remove it?

Reply:
Yeah I wasn't sure about this one, either. I think it's worth keeping if someone fills in the details regarding design principles but not otherwise.

Reply:
I'll remove it for simplicity